<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>530</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSortName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Trier par nom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSortYear">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Trier par date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSortType">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>530</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Trier par type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemove">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHTML">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>97</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Exporter
en
HTML</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
